### ISSUE TITLE
release 0.2

### ISSUE THREAD

**Issue by perrygeo** [mapbox/rio-color#30]
When the [0.2 milestone](https://github.com/mapbox/rio-color/milestones/0.2) is wrapped up, it's time to publish 0.2.

This time, I need to straighten out a few issues
* [x] check on best practices for distributing cython
* [x] How to efficiently build binary wheels for as many platforms/py versions as reasonable
* [x] Upload with twine
* [ ] Build linux wheels & publish to private package indexes (possibly digging into manylinux binaries depending on https://github.com/mapbox/rasterio/issues/693)

In short, I just need to get a good release workflow down and documented.

@sgillies can we chat a bit about this next week?
